shibata added English to some eadme.md and translated vantiq-install-closed-network-aws.md and fluentd.md (#24)
fujitake/vantiq-related: vantiq-apps-development/1-day-workshop/docs/eng/a10-dev01_detect_congestion_app.md

>     * Can do simple filtering, or can be flexible decision making by configuring `Procedure` as threshold.

Reviewer: ```suggestion
    * Can do simple filtering, or can do more complex filtering by leveraging a `Procedure`.
```

  * `AccumulateState`

>     * Since events received one after another can be tracked and counted, depending on how the `Procedure` is described, it can be implemented with a high flexibility, such as calculation using the number of events received in a certain period of time.

Reviewer: ```suggestion
    * The task is capable of tracking the state across consecutive events. Also combining with `Procedure`, the task can implement logics such as calculation using the number of events received in a certain period of time.
```
